python/ray/ml/train/integrations/huggingface/huggingface_trainer.py

```diff
@@ -32,15 +101,23 @@ class HuggingFaceTrainer(TorchTrainer):
     shards, with each Actor training on a single shard.
     All the other datasets will not be split.
 
+    The datasets will NOT be shuffled by default. Call ``Dataset.random_shuffle()``
+    on the "train" dataset you are passing in ``datasets`` if you wish for the
+    training data to be shuffled.
```

Comment:
I don't think we should recommend this. This will force an expensive global shuffle. Can we at least recommend using local TF/torch-specific shuffles after gather the data locally?

cc @richardliaw @gjoliver @matthewdeng

Reply:
I can add random local shuffle in by default.